src/main/groovy/life/qbic/datamodel/accounting/CostEstimate.groovy

```diff
@@ -1,5 +1,7 @@
 package life.qbic.datamodel.accounting
 
+import life.qbic.datamodel.dtos.business.CostEstimateId
+import life.qbic.datamodel.dtos.business.QuotationId
```

Comment:
Just to note that down here:
at this point, the objects become business objects and are not simply DTOs anymore.